### PR TITLE
Nando PR post

### PR DESCRIPTION
**Pull request by kmaida**
PR post for Nando's choosing Auth0. Publish tomorrow 4/21

### CODE REVIEW

File: _posts/2017-04-21-nandos-chooses-auth0-to-centralize-identity-management.markdown [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+- press
+- auth0
+- nandos
+related:
+- 2017-03-31-polaris-chooses-auth0-for-b2b-b2c-b2e
+- 2017-02-27-houghton-mifflin-harcourt-chooses-auth0-for-identity-management
+- 2016-12-23-auth0-named-in-seattle-10-hottest-ventures-2016
+---
+
+**BELLEVUE, WA** - [Nando's](https://www.nandos.com/), a Portugese chicken restraurant with South African roots, has chosen Auth0 as their identity management platform for internal and public-facing applications.
```
Comment (brunokrebs) [inline _posts/2017-04-21-nandos-chooses-auth0-to-centralize-identity-management.markdown]: `restraurant` <= typo

File: _posts/2017-04-21-nandos-chooses-auth0-to-centralize-identity-management.markdown [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+Auth0 gives Nando's the robust Identity and Access Management (IAM) solution they need to integrate with applications both within and outside their own ecosystem. Internally, Nando's utilizes a cloud platform for project management, meeting software, AWS, JIRA, and more. With the increasing distribution of these applications, managing many identity providers and provisioning users was a challenge that Auth0 overcomes for Nando's. In addition, Nando's employs over 14,000 workers across 1300 restaurants in five countries. Using an app to engage on a more personal level and share information with their workers is also an important need for Nando's. A seamless user experience, Single Sign-On, and account linking were among the key aspirations when considering an IAM solution and selecting Auth0.
```
Comment (brunokrebs) [inline _posts/2017-04-21-nandos-chooses-auth0-to-centralize-identity-management.markdown]: `was a challenge that Auth0 overcomes for Nando's` <= wouldn't it be `overcame` or `is a challenge`?

File: _posts/2017-04-21-nandos-chooses-auth0-to-centralize-identity-management.markdown [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+### About Auth0
+
+Auth0 provides frictionless authentication and authorization for developers. The company makes it easy for developers to implement even the most complex identity solutions for their web, mobile, and internal applications. Ultimately, Auth0 allows developers to control how a person’s identity is used with the goal of making the internet safer. As of August, 2016, Auth0 has raised over $24m from Trinity Ventures, Bessemer Venture Partners, K9 Ventures, Silicon Valley Bank, Founders Co-Op, Portland Seed Fund and NXTP Labs, and the company is further financially backed with a credit line from Silicon Valley Bank.
+
+For more information visit [https://auth0.com](https://auth0.com) or follow [@auth0](https://twitter.com/auth0) on Twitter.
```
Comment (brunokrebs) [inline _posts/2017-04-21-nandos-chooses-auth0-to-centralize-identity-management.markdown]: `or follow` <= `and follow` 😄

File: _posts/2017-04-21-nandos-chooses-auth0-to-centralize-identity-management.markdown (hunk 1 above)
Comment (kmaida) [inline _posts/2017-04-21-nandos-chooses-auth0-to-centralize-identity-management.markdown]: 👍 Nice catch, thanks

File: _posts/2017-04-21-nandos-chooses-auth0-to-centralize-identity-management.markdown (hunk 2 above)
Comment (kmaida) [inline _posts/2017-04-21-nandos-chooses-auth0-to-centralize-identity-management.markdown]: `is a challenge that Auth0 overcomes`

File: _posts/2017-04-21-nandos-chooses-auth0-to-centralize-identity-management.markdown (hunk 3 above)
Comment (kmaida) [inline _posts/2017-04-21-nandos-chooses-auth0-to-centralize-identity-management.markdown]: Leaving as-is. We don't necessarily need to instruct the user to visit the site AND follow us on Twitter. (Also this is a copy-paste from all PR posts.)

**Pull request merged**